src/Neuron/Zettelkasten/View.hs

```diff
@@ -144,7 +173,11 @@ renderTags tags = do
     -- the top pushes the zettel content down, introducing unnecessary white
     -- space below the title.
     span_ [class_ "ui black right ribbon label", title_ "Tag"] $ do
-      toHtml @Text tag
+      a_
+        [ href_ (Rib.routeUrlRel $ Route_Search Nothing [tag]),
```

Comment:
Oh, and here it would look like:
```
Rib.routeUrlRel Route_Search <> "?tag=" <> tag ...
```
(Would be nice to add a function to Rib.Route that returns something like `Text.URI` instead).